### ISSUE TITLE
Two Sass files, only the first one gets purged

### ISSUE THREAD

**Issue by alesvaupotic** [spatie/laravel-mix-purgecss#24]
This config
```
mix
  .js("resources/assets/js/app.js", "public/js")
  .sass("resources/assets/css/main.scss", "public/css")
  .sass("resources/assets/css/auth.scss", "public/css")
  .purgeCss()
  ...
```
produces only one purged file:
```
        Asset     Size  Chunks             Chunk Names
   /js/app.js   202 kB       0  [emitted]  /js/app
/css/main.css  8.85 kB       0  [emitted]  /js/app
/css/auth.css   230 kB    0, 0  [emitted]  /js/app, /js/app

```

Is this intentional? Because it sure is not expected. If you swap the order of Sass files, again the first one gets purged, the second one is ignored.

**Comment by cjmaxik** [comment]
Should be resolved in v2.0 (#23)
![image](https://user-images.githubusercontent.com/1501705/37763284-56175d2c-2e0a-11e8-8d77-73a26128f0a4.png)

**Comment by alesvaupotic** [comment]
Works as advertised. 

```
              Asset     Size  Chunks             Chunk Names
         /js/app.js   107 kB       0  [emitted]  /js/app
/js/x-components.js   3.4 kB       1  [emitted]  /js/x-components
      /css/main.css  13.2 kB       0  [emitted]  /js/app
      /css/auth.css  7.59 kB    0, 0  [emitted]  /js/app, /js/app

```

**Comment by alesvaupotic** [comment]
Solved in 2.0.0.
